lib/studies/reply/mail_card_preview.dart

```diff
@@ -47,11 +51,87 @@ class MailPreviewCard extends StatelessWidget {
           message: email.message,
           onTap: openContainer,
         );
+
+        if (isDesktop) {
+          return mailPreview;
+        } else {
+          return Dismissible(
+            key: ObjectKey(email),
+            dismissThresholds: const {
+              DismissDirection.startToEnd: 0.4,
+              DismissDirection.endToStart: 1,
+            },
+            onDismissed: (direction) {
+              switch (direction) {
+                case DismissDirection.endToStart:
+                  break;
+                case DismissDirection.startToEnd:
+                  break;
+                default:
+              }
+            },
+            background: _DismissibleContainer(
+              icon: 'twotone_delete',
+              backgroundColor: colorScheme.primary,
+              iconColor: ReplyColors.blue50,
+              alignment: Alignment.centerLeft,
+              padding: const EdgeInsetsDirectional.only(start: 20),
+            ),
+            secondaryBackground: _DismissibleContainer(
+              icon: 'twotone_star',
+              backgroundColor: colorScheme.secondary,
+              iconColor: ReplyColors.black900,
+              alignment: Alignment.centerRight,
+              padding: const EdgeInsetsDirectional.only(end: 20),
+            ),
+            child: mailPreview,
+          );
+        }
       },
     );
   }
 }
 
+class _DismissibleContainer extends StatelessWidget {
+  const _DismissibleContainer({
+    @required this.icon,
+    @required this.backgroundColor,
+    @required this.iconColor,
+    @required this.alignment,
+    @required this.padding,
+  })  : assert(icon != null),
+        assert(backgroundColor != null),
+        assert(iconColor != null),
+        assert(alignment != null),
+        assert(padding != null);
+
+  final String icon;
+  final Color backgroundColor;
+  final Color iconColor;
+  final Alignment alignment;
+  final EdgeInsetsDirectional padding;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: backgroundColor,
+      alignment: alignment,
+      padding: padding,
+      child: Material(
```

Comment:
Can Material be removed? It doesn't look necessary

Reply:
Without it using `ImageIcon` with a custom asset over an `Icon` results in an offset like below.

![Screen Shot 2020-08-17 at 9 17 56 PM](https://user-images.githubusercontent.com/948037/90470584-bdc75c80-e0d0-11ea-9a68-d83fcaea669a.png)

Reply:
Do you know if there a framework bug filed for that?

Reply:
I filed this https://github.com/flutter/flutter/issues/63229 when I thought the issue was isolated to the navigation rail. Should I make a new issue or edit the one I already made?

Reply:
Could you check if this occurs in light mode as well?

Reply:
Yup, here it is in light mode in two different spots.

![Screen Shot 2020-08-20 at 8 08 05 AM](https://user-images.githubusercontent.com/948037/90793336-78698180-e2c0-11ea-9cf9-fb6773262414.png)
![Screen Shot 2020-08-20 at 8 08 25 AM](https://user-images.githubusercontent.com/948037/90793346-7acbdb80-e2c0-11ea-976d-2f8be82a99b7.png)